tests/x11/flatpak.pm

```diff
@@ -32,8 +32,8 @@ sub run {
     die "flatpak list is not empty" if script_output("flatpak list") != "";
     assert_script_run('flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo');
     assert_script_run('flatpak search gimp | grep -i "org.gimp.GIMP"');
-    assert_script_run('flatpak search steam | grep -i "com.valvesoftware.Steam"');
-    assert_script_run('flatpak install -y org.gimp.GIMP');
+    assert_script_run('flatpak search vlc | grep -i "org.videolan.VLC"');
+    assert_script_run('flatpak install -y org.gimp.GIMP', timeout => 300);
```

Comment:
does the timeout actually required here. For the commit message i am not sure why this change happened.

Reply:
Yes it is, some VRs run into timeouts there